src/wallet/rpcdump.cpp

```diff
@@ -535,7 +556,14 @@ UniValue importwallet(const JSONRPCRequest& request)
     file.close();
     pwallet->ShowProgress("", 100, std::function<void(void)>()); // hide progress dialog in GUI
     pwallet->UpdateTimeFirstKey(nTimeBegin);
-    pwallet->RescanFromTime(nTimeBegin, false /* update */);
+    int64_t scanned_time = pwallet->RescanFromTime(nTimeBegin, false /* update */);
+    if (scanned_time > TIMESTAMP_MIN) {
```

Comment:
In commit "Give an error when rescan is aborted by the user"

s/TIMESTAMP_MIN/nTimeBegin/ here to fix test failures

Reply:
Done